modules/acquia_cms_site_studio/acquia_cms_site_studio.info.yml

```diff
@@ -5,11 +5,11 @@ core_version_requirement: ^9.5 || ^10
 type: module
 dependencies:
   - acquia_cms_common:acquia_cms_common
-  - cohesion_base_styles
-  - cohesion_custom_styles
-  - cohesion_elements
-  - cohesion_style_helpers
-  - cohesion_sync
-  - cohesion_website_settings
+  - cohesion_base_styles # phpcs:ignore
```

Comment:
Use at the top of the file
# phpcs:ignoreFile

Reply:
That will ignore **ALL** PHPCS Rules across the file, but we want to ignore only the specific lines. That's why we have not added at the top.